cluster-provision/gocli/cmd/run.go

```diff
@@ -557,6 +561,15 @@ func run(cmd *cobra.Command, args []string) (retErr error) {
 		}
 	}
 
+	sharedVolumeName := prefix + "-shared"
+	if len(sharedDisks) > 0 {
+		sharedVolume, err := cli.VolumeCreate(ctx, volume.CreateOptions{Name: sharedVolumeName})
```

Comment:
rm.go (cluster-down) will clean up this volume. Technically any volume with the prefix